models/reddit_ads__account_report.sql

```diff
@@ -26,6 +26,7 @@ accounts as (
         sum(report.spend) as spend
 
         {{ fivetran_utils.persist_pass_through_columns(pass_through_variable='reddit_ads__account_passthrough_metrics', transform = 'sum') }}
+        
```

Comment:
To be consistent with our style standards, would you mind providing an additional space before the `from` following the macro call. Could you then apply the same to the remaining models.

Reply:
done!